app/controllers/spree/users_controller.rb

```diff
@@ -31,13 +30,10 @@ def registered_email
       registered = Spree::User.find_by(email: params[:email]).present?
 
       if registered
-        render status: :ok, cable_ready: cable_car.
-          inner_html(
-            "#login-feedback",
-            partial("layouts/alert",
-                    locals: { type: "alert", message: t('devise.failure.already_registered') })
-          ).
-          dispatch_event(name: "login:modal:open")
+        respond_to do |format|
+          format.html { head :ok }
+          format.turbo_stream { :registered_email }
```

Comment:
This change isn't accompanied with a change in users_controller_spec. We need to add `format: :turbo_stream`.

I was expecting it to fail, because you forgot to write `render:` before `:registered_email`. But Rails' default handling kicks in and renders it anyway ;) 
Because it's simply rendering a partial of the same name, we don't need the block at all:

```suggestion
          format.turbo_stream
```

Reply:
Hmm, this isn't covered by a system spec. 
In fact, on closer look, I can see it's not even used!  After some git detective work, I managed to find out it hasn't been used for over a year, since https://github.com/openfoodfoundation/openfoodnetwork/commit/fc0cca2210dcd78f4554fea0dc76af712c05922c#diff-96984374385f56a535a566ac8afee14de6461121ed5717e2ddced0e640d2c75b 

So we can remove this action entirely!

Reply:
Thanks
Nice one spotting this one